.github/workflows/nightly.yml

```diff
@@ -437,6 +437,26 @@ jobs:
         uses: ./.github/actions/notify-slack
         with:
           token: ${{ secrets.ACTION_MONITORING_SLACK }}
+      - name: Test Drupal
+        if: always()
+        run: |
+          php -i|grep memory_limit
+          echo memory_limit=256M > /etc/php.d/drupal.ini
```

Comment:
This should be done in setup, as it will leak into the next step anyway. Increasing the memory limit generally should be ok.

Reply:
Maybe this change could go separate file which will be removed at the end of script?